loaders/express.js

```diff
@@ -39,7 +41,7 @@ const loader = async app => {
   app.use(cors())
 
   // near the top with the other imports
-
+  //following from bob zirolls tutorial
```

Comment:
Hi @cynthiaeddy! Do you think it could be helpful to add the link to the listed resource for easy future reference?

Reply:
done!